### DISCUSSION TITLE
Plans for scikit-learn 1.4 compatible release

### DISCUSSION THREAD

**Discussion by celestinoxp** [alan-turing-institute/sktime#5831] [General]
@fkiraly Hello! I'm currently working on a PR to add support for scikit-learn 1.4 in PyCaret. As sktime is a direct dependency on PyCaret for time series functionality, I would like to know what the forecast is for the release of a new version of sktime with support for scikit-learn 1.4. This update is crucial to avoid bugs in PyCaret. Thank you for your attention to this matter.

**Comment by fkiraly** [comment]
The plan is bumping the bound in 0.26.0 which is scheduled for today-ish. The current dev version (`main` branch) should be fully 1.4 compatible, except for the `pyproject` bound.

There is somewhat of an ambition to also fix problems with `pandas` 2.2.0 for 0.26.0, though that may prove more problematic.
